docs/courses/coursemanagement.md

```diff
@@ -58,7 +58,7 @@ Please [click here](/courses/authoring/#project-course-assignment-or-book) for d
 This describes the first time you create a new assignment.
 
 1. **Edit your assignment**. If your assignment is a project assignment, clicking the assignment will take you into the IDE, as with a Codio project, where you can work on your project.
-1. **Publish your assignment**. When you think your assignment is in good shape, it is ready to publish. A assignment is not available to the course until you publish. At this point a snapshot is taken of the assignment and it forms a part of the published course.
+1. **Publish your assignment**. When you think your assignment is in good shape, it is ready to publish. A assignment is not available to the course until you publish. At this point a snapshot is taken of the assignment and it forms a part of the published course. 
```

Comment:
"When you think your assignment is ready to be deployed to students, it is ready to publish. An assignment is not available to the course until you publish it. During the publishing process, a snapshot is taken of the assignment and it forms a part of the published course."